public/app/features/dashboard-scene/settings/variables/editors/AdHocFiltersVariableEditor.tsx

```diff
@@ -52,6 +58,54 @@ export function AdHocFiltersVariableEditor(props: AdHocFiltersVariableEditorProp
       defaultKeys={defaultKeys}
       onDefaultKeysChange={onDefaultKeysChange}
       onAllowCustomValueChange={onAllowCustomValueChange}
+      inline={props.inline}
     />
   );
 }
+
+export function getAdHocFilterOptions(variable: SceneVariable): OptionsPaneItemDescriptor[] {
+  if (!(variable instanceof AdHocFiltersVariable)) {
+    console.warn('getAdHocFilterOptions: variable is not an AdHocFiltersVariable');
+    return [];
+  }
+
+  return [
+    new OptionsPaneItemDescriptor({
+      render: () => <ModalEditor variable={variable} />,
```

Comment:
why a modal editor? the adhoc filters options can all exist in the side pane

Reply:
really to just display the code editor.  wasn't sure if folks were pasting csv files in there, how big that list would get and how much room was needed.  wanted to also avoid scrollbars within scrollbars.  the code editor has it's own scrollbar

sounds like not much room is needed so will change this to render inline

Reply:
@torkelo updated .... the description has the new screenshot.